volttron/platform/certs.py

```diff
@@ -503,8 +503,8 @@ def create_csr(self, fully_qualified_identity, remote_instance_name):
         The key that is used to sign the csr is <instance_name>.name.
 
         :param fully_qualified_identity:
-        :param target_volttron:
-        :return:
+        :param remote_instance_name:
+        :return csr.public_bytes:
```

Comment:
So what exactly is csr.publich_bytes...https://cryptography.io/en/latest/hazmat/primitives/asymmetric/serialization/?highlight=public_bytes#cryptography.hazmat.primitives.serialization.PublicFormat

Reply:
It looks to be an encoded form of the certificate, which can be stored or transmitted. Updated docstring to better reflect this.